Yosemite/Yosemite/Model/Updaters/ProductUpdater.swift

```diff
@@ -190,7 +190,7 @@ extension Product: ProductUpdater {
                        dimensions: dimensions,
                        shippingRequired: shippingRequired,
                        shippingTaxable: shippingTaxable,
-                       shippingClass: shippingClass?.slug,
+                       shippingClass: shippingClass?.slug ?? "",
```

Comment:
There is a reason why we return now an empty string in case the slug is null?

Reply:
the `shippingClass` API field (`shipping_class`) is empty when there's no shipping class set in core. when we update a `Product` with a `nil` shipping class object, if we don't default to empty then the `Product` would be different from the original. Without this line, some tests would fail because the initial `shippingClass` field is empty from the API response.